ipfx/feature_vectors.py

```diff
@@ -97,21 +97,53 @@ def extract_feature_vectors(data_set,
     return all_features
 
 
-def extract_multipatch_feature_vectors(lsq_sweeps, lsq_start, lsq_end,
+def extract_multipatch_feature_vectors(lsq_supra_sweeps, lsq_supra_start, lsq_supra_end,
```

Comment:
@sgratiy I’m happy to add tests. I’m not sure if unit tests are the most helpful here, so it would be more of evaluating with test data and comparing to example output. I’m not sure how you have that infrastructure set up. For example, it doesn’t seem like the files for comparison are in the repository?